src/AssetsController.php

```diff
@@ -62,9 +62,7 @@ public function register_assets() {
 		// The price package is shared externally so has no blocks prefix.
 		$this->api->register_script( 'wc-price-format', 'build/price-format.js', [], false );
 
-		if ( Package::feature()->is_feature_plugin_build() ) {
-			$this->api->register_script( 'wc-blocks-checkout', 'build/blocks-checkout.js', [] );
-		}
```

Comment:
The reason for this removal is: Mini Cart Contents inner blocks use `registerCheckoutBlock()` from `@woocommerce/blocks-checkout` which is mapped to `wc-blocks-checkout`.

@ralucaStan Is there any issue if we make this script available in WC Core?

Reply:
Without a lot of background around this, my thoughts go around two things:
- would it be a performance issue to add all the content of `@woocommerce/blocks-checkout`, assuming that the MiniCart doesn't use all the exports from the package?
- I see that `@woocommerce/blocks-checkout` also exports this: `export { SlotFillProvider } from 'wordpress-components';` @Aljullu do you know why this is the case?

Reply:
> - would it be a performance issue to add all the content of `@woocommerce/blocks-checkout`, assuming that the MiniCart doesn't use all the exports from the package?

That's a good point, @ralucaStan! However, I don't know how much we could reduce the bundle size. Even though Mini Cart only imports `registerCheckoutBlock()` directly, it includes some Cart components that have other dependencies which are required by extensions that want to apply filters (ie: WC Subscriptions).

> - I see that `@woocommerce/blocks-checkout` also exports this: `export { SlotFillProvider } from 'wordpress-components';` @Aljullu do you know why this is the case?

Unfortunately, I don't recall it. @senadir do you have any clue? Could it be deprecated and import `SlotFillProvider` from `@wordpress/components` instead?